bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TTZHTo4b_5f_LO/TTZHTo4b_5f_LO_customizecards.dat

```diff
@@ -0,0 +1,4 @@
+#put card customizations here (change top and higgs mass for example)
+set param_card mass 6 172.5
+set param_card yukawa 6 172.5
+set param_card mass 25 125.0
```

Comment:
Same comment as in PR https://github.com/cms-sw/genproductions/pull/3189 , I don't think this card is actually needed in this case as well